crates/aide/Cargo.toml

```diff
@@ -31,8 +31,8 @@ cfg-if = "1.0.0"
 # custom axum extractors
 serde_qs = { version = "0.12.0", optional = true }
 jwt-authorizer = { version = "0.9", optional = true }
-axum-sqlx-tx = { version = "0.5", optional = true }
-sqlx = { version = "0.6", optional = true }
+axum-sqlx-tx = { git = "https://github.com/Wicpar/axum-sqlx-tx.git", optional = true }
```

Comment:
What's up with this pointing at your own custom fork, are you planning to maintain that library in the future?

Reply:
It's obviously a mistake, i need to create a proper build using my fork and cannot wait on the library accepting the changes or not.
I'll redo a proper PR when i have the time.